memcache/memcache.go

```diff
@@ -181,6 +183,14 @@ type conn struct {
 	c    *Client
 }
 
+// BytesPool is a pool of bytes that can be reused.
+type BytesPool interface {
+	// Get returns a new byte slice that fits the given size.
```

Comment:
"fits the given size" seems a bit wooly; could we say something like as "where `cap` >= sz" ?